dist/main/index.js

```diff
@@ -14536,8 +14537,15 @@ function ownerRepo(url) {
   };
 }
 
-async function genhtml(coverageFiles, tmpPath) {
-  const workingDirectory = core.getInput('working-directory').trim() || './';
+function ensureTrailingSlash(path) {
+  if (path.endsWith("/")) {
+    return path;
+  }
+
+  return path + "/";
+}
+
+async function genhtml(coverageFiles, tmpPath, workingDirectory) {
   const artifactName = core.getInput('artifact-name').trim();
   const artifactPath = path.resolve(tmpPath, 'html').trim();
   const args = [...coverageFiles];
```

Comment:
minor js style, `concat` allows this to be one line, with no mutations:

```
const args = [...coverageFiles].concat(['--output-directory', artifactPath])
```

Reply:
Turns you you can also do `[..coverageFiles, '--output-directory', artifactPath]`!

Reply:
of course -- even better!

Reply:
You are probably aware, but JS has this syntax for objects which is similar to Elixir, but I haven't noticed in our code too often in the past:

```
const someObj  = {...anotherObj, abc: "def", [someVar]: "qrs"};
```